inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/ModalPanel.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020
```

Comment:
The class name is not saying very much about the purpose of the class. Maybe something like `WorkloadItemDetailDialogContent`?